madgraph_production_tools/run_production.py

```diff
@@ -9,7 +9,8 @@
 import subprocess
 import physics as ph
 
-username = os.getlogin()
```

Comment:
If we could make this work it might be nice to do something similar for the manually set paths in the limit tools

Reply:
I think there are a few small "quality of life" improvements we could implement (figuring out paths more automatically etc.). Maybe we leave it as is for now and then when we move to CMS analysis we first spend some time improving things like that?

Reply:
Yes agreed :)